force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +418,13 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
+        if (person1.getAge() > person2.getAge()) {
+            return person1;
+        } else if (person1.getAge() < person2.getAge()) {
+            return person2;
+        } else if (person1.getAge() == person2.getAge()) {
+            return person1;   
+        }
```

Comment:
As a challenge, can you simplify this? If the values are tied, you can return them either. You don,t need any else conditions